automatic/tightvnc/tightvnc.nuspec

```diff
@@ -29,8 +29,4 @@ With TightVNC, you can:
     <tags>remote control vnc admin</tags>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
   </metadata>
-  <files>
-    <file src="tools\**" target="tools" />
-  </files>
-</package>
-<!-- “UTF-8” test -->
```

Comment:
Don't remove the UTF-8 tests, either add this back, or add the UTF-8 test created by the default `choco new` command.
Also don't remove the files element, this could potentially make choco pack other files located in the same folder.